Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/11-blue-green.md

```diff
@@ -0,0 +1,271 @@
+  - A record mapped with the PIP of the Blue Application Gateway
+3. T2: Sync K8S State between Blue and Green clusters
+4. T3: Traffic Switch to the green cluster
+  - "blue cluster" and "blue app gateway" with aks_turn_on=true and appgw_turn_on=true
+  - "green cluster" and "green app gateway" with aks_turn_on=true and appgw_turn_on=true
+  - A record mapped with the PIP of the Green Application Gateway
+5. T4: Blue cluster is destroyed
+  - "blue cluster" and "blue app gateway" with aks_turn_on=false and appgw_turn_on=false
+  - "green cluster" and "green app gateway" with aks_turn_on=true and appgw_turn_on=true
+  - A record mapped with the PIP of the Green Application Gateway
+
+## T0: Blue Cluster is On
+
+  Follow the guide [Getting Started with the default values](../README.md). Where the default values are:
+
+- in the file "Scenarios\AKS-Secure-Baseline-PrivateCluster\Terraform\05-Network-LZ\app-gateway.tf"
+```
+locals {
+  appgws = {
+    "appgw_blue" = {
+      name_prefix="blue"
+      appgw_turn_on=true
+    },
+    "appgw_green" = {
+      name_prefix="green"
+      appgw_turn_on=false
+    }
+  }
+}
+
+```
+
+- in the file "Scenarios\AKS-Secure-Baseline-PrivateCluster\Terraform\07-AKS-cluster\aks-cluster.tf"
+
+```
+locals {
+  aks_clusters = {
+    "aks_blue" = {
+      name_prefix="blue"
+      aks_turn_on=true
+      k8s_version="1.23.5"
+      appgw_name="lzappgw-blue"
+    },
+    "aks_green" = {
+      name_prefix="green"
+      aks_turn_on=false
+      k8s_version="1.23.5"
+      appgw_name="lzappgw-green"
+    }
+  }
+}
+
+```
+
+After the deployment if the Landing Zone, install a sample application to test the deployment. The sample application to use is stored in the file "Scenarios\AKS-Secure-Baseline-PrivateCluster\Terraform\07-AKS-cluster\sample-workload-for-agic-test.yaml".
+
+```bash
+az aks command invoke --resource-group $ClusterRGName --name $ClusterName   --command "kubectl apply -f sample-workload-for-agic-test.yaml "
+```
+
+As an alternative you can run the kubectl command directly from the Linux jump box access of the BastionHost provisoned as part in the [hub network](./04-network-hub.md).
+
+If you have deployed the public dns zone that is part of [AKS-Supporting](./06-aks-supporting.md), than you can test the deployment, executing an invocation to the sample app.
+
+```bash
+curl http://{hostname-app}.{public_domain}/
+```
+
+If the azure public dns zone is not attached to the domain, than is possible to test the app endpoint with the followwing command.
+
+```bash
+curl -H "Host: {hostname-app}.{public_domain}" http://{app-gateway-pip}/
+```
+
+## T1: Green Cluster Deployment
+
+At this stage is required to perfor the following action to deploy the new green cluster in co-existence with the blue one.
+
+1. Run again the flow mentioned [here](./05-network-lz.md), with the following configuration in the file "Scenarios\AKS-Secure-Baseline-PrivateCluster\Terraform\05-Network-LZ\app-gateway.tf"
```

Comment:
Instead of referring to the other article, can we instead simply say:
Go to the folder ./05-Network-LZ, customize the file xxx and run "terraform apply".

Or do they also have to also run terraform init and terraform plan?